src/views/states/list/components/InterfacesTypeSection.tsx

```diff
@@ -75,17 +76,17 @@ const InterfacesTypeSection: FC<InterfacesTypeSectionProps> = memo(
                 )}
               </Td>
               <Td>
-                {iface.bridge?.port?.length ? (
+                {ports?.length ? (
                   <Tooltip
                     content={
                       <List isPlain>
-                        {iface?.bridge?.port.map((portItem) => (
-                          <ListItem key={portItem.name}>{portItem.name}</ListItem>
+                        {ports.map((port) => (
+                          <ListItem key={port}>{port}</ListItem>
                         ))}
                       </List>
                     }
                   >
-                    <span> {iface?.bridge?.port?.length}</span>
+                    <span> {ports.length}</span>
```

Comment:
Hmm, do we need that extra empty space here before `{ports.length}`?

Reply:
No :-)